scripts/monitoring/cron-send-os-master-metrics.py

```diff
@@ -52,6 +57,9 @@ def run(self):
         if self.args.user_count or self.args.all_checks:
             self.user_count()
 
+        if self.args.metrics or self.args.all_checks:
+            self.metric_check()
+
         self.zagg_sender.send_metrics()
```

Comment:
* Group these into 3 try/catch groups. 
  - API
  - Metrics
  - Health
* Remove catches from inside the functions and let the outer exception catch the problems and send the zabbix ping item values.